Description: add $Config{ccflags} to CCFLAGS to fix FTBFS with perl 5.14
Origin: vendor
Bug-Debian: http://bugs.debian.org/638894
Forwarded: not-needes
Author: Dominic Hargreaves <dom@earth.li>
Reviewed-by: gregor herrmann <gregoa@debian.org>
Last-Update: 2011-09-23

--- a/ext/Module/Install/PRIVATE/Mongo.pm
+++ b/ext/Module/Install/PRIVATE/Mongo.pm
@@ -1,5 +1,6 @@
 use strict;
 use warnings;
+use Config;
 
 package Module::Install::PRIVATE::Mongo;
 
@@ -17,6 +18,10 @@
     my ($self, @mongo_vars) = @_;
     my $custom_cflags = 0;
     my $ccflags = $self->makemaker_args->{CCFLAGS};
+
+    # #638894
+    $ccflags .= " $Config{ccflags}");
+    $custom_cflags = 1;
 
     if ($Config{osname} eq 'darwin') {
         my @arch = $Config::Config{ccflags} =~ m/-arch\s+(\S+)/g;
